tools/crm_mon_output.c

```diff
@@ -444,14 +444,16 @@ cluster_dc_xml(pcmk__output_t *out, va_list args) {
 }
 
 static int
-cluster_options_html(pcmk__output_t *out, va_list args) {
+cluster_options_html_text(pcmk__output_t *out, va_list args) {
```

Comment:
Despite being very close, I like the freedom of having the formats separate so each can be optimal rather than both acceptable

Reply:
Heh, okay.  They are just really, really similar.  But I can do that.